react/components/SearchBar/components/SearchBar.js

```diff
@@ -38,10 +39,18 @@ export default class SearchBar extends Component {
       />
     )
 
-    const mainClasses = classNames(styles.searchBarContainer)
+    const mainClasses = `w-100 mw7 pv4 ${classNames(styles.searchBarContainer)}`
```

Comment:
```suggestion
    const mainClasses = classNames('w-100 mw7 pv4', styles.searchBarContainer)
```

This and I would also just inline it in the div, like: `<div className={classNames(..`